README.md

```diff
@@ -57,6 +57,7 @@ Secrets which have been applied to the prow cluster but are not committed here a
 - `GitHub` secrets:
  - `bot-token-github` in the default namespace
  - `bot-token-github` in the github-admin namespace
+ - `ouath-token` in the default namespace
```

Comment:
would it be worth mentioning that these are "personal access tokens" specifically?  (Or is this already mentioned elsewhere and I missed it?) I got a bit lost poking around the github UI looking for "secrets", which do exist but are a different concept in Github land apparently.